integration-test/src/main/resources/testsuites/e2e/aws-longrunning-e2e-tests.yaml

```diff
@@ -9,4 +9,5 @@ tests:
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.distrox.ephemeral.DistroXStopStartTest
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.distrox.ephemeral.DistroXRepairTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.distrox.ephemeral.DistroXUpgradeTests
-      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRepairTests
\ No newline at end of file
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRepairTests
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxResizeTests
```

Comment:
This is a duplicated test reference for the `com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxResizeTests`. If the test execution time is less than 150 minutes, please remove this line from here.

Reply:
addressed.